sdks/python/scripts/run_pylint.sh

```diff
@@ -64,6 +64,11 @@ while IFS= read -d $'\0' -r file ; do
     EXCLUDED_GENERATED_FILES+=("$file")
 done < <(find apache_beam/portability/api -type f -name "*pb2*.py" -print0)
 
+# Ignore vendor files.
+while IFS= read -d $'\0' -r file ; do
+    EXCLUDED_GENERATED_FILES+=("$file")
```

Comment:
this is no longer "generated files", let's pick a better name / new variable.

Reply:
yeah, EXCLUDED_FILES seems better.